gm4_garden_variety/data/gm4_garden_variety/functions/integration/fruiting_trees/update_leaves.mcfunction

```diff
@@ -0,0 +1,7 @@
+# checks whether @s is a fruit leaf, and if so allows further evaluation of the age
+# @s = @e[type=armor_stand,tag=gm4_fruiting_leaf]
+# at @s align xyz
+# run from #gm4_fruiting_trees:tree/leaf/fruiting/age
```

Comment:
From what I can tell this function isn't called from anywhere. Were you planning to integrate this with fruiting trees?